src/transform/create-transform.ts

```diff
@@ -68,28 +68,22 @@ export default ({ browser }: { browser: boolean }) => {
           const excludeDirs = taglibConfig.excludeDirs;
           if (excludeDirs) {
             for (const name of excludeDirs) {
-              compiler.taglibFinder.excludeDir(name);
+              compiler.taglib.excludeDir(name);
             }
           }
 
           const excludePackages = taglibConfig.excludePackages;
           if (excludePackages) {
             for (const name of excludePackages) {
-              compiler.taglibFinder.excludePackage(name);
+              compiler.taglib.excludePackage(name);
             }
           }
         }
       }
 
-      const result = compiler[
-        (browser || (config as any).browser) &&
-        compiler.compileForBrowser /** Only Marko 4 supports compileForBrowser, otherwise use compile */
-          ? "compileForBrowser"
-          : "compile"
-      ](src, filename, MARKO_OPTIONS);
-
+      const result = compiler.compileSync(src, filename, MARKO_OPTIONS);
```

Comment:
This is definitely the biggest unknown; there is no `compileForBrowser` in Marko 5